public_api_client/public_api_client/configuration.py

```diff
@@ -0,0 +1,9 @@
+import json
+import os
+
+config_file_path = os.getenv("PUBLIC_API_CONFIG_FILE_PATH", "/home/pib/public_api/config.json")
+with open(config_file_path) as f: _data = f.read()
+_config = json.loads(_data)
+
+tryb_url_prefix = _config["trybUrlPrefix"]
```

Comment:
Is it possible to move this into the credentials folder?
Or create new credential files linked here.